src/main/java/uk/gov/ida/eventemitter/EventEmitterModule.java

```diff
@@ -15,12 +23,23 @@ protected void configure() {}
 
     @Provides
     private Optional<AmazonSQS> getAmazonSqs(final Optional<Configuration> configuration) {
-        if (configuration.isPresent() && configuration.get().getSourceQueueName() != null) {
+        if (configuration.isPresent() &&
+            configuration.get().getSourceQueueName() != null) {
             return Optional.ofNullable(AmazonSQSClientBuilder.defaultClient());
         }
         return Optional.empty();
     }
 
+    @Provides
+    private Optional<AmazonS3> getAmazonS3(final Optional<Configuration> configuration) {
+        if (configuration.isPresent() &&
+            configuration.get().getBucketName() != null &&
+            configuration.get().getKeyName() != null) {
```

Comment:
Perhaps it's time to either refactor this logic out into a `isUsingAWS` method, or to refactor the config to include a `useRealSqs` switch.